### PR TITLE
Remove ast.literal_eval from install_packs

### PR DESCRIPTION
**Pull request by ilaner**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5890?filter=-2

## Description
No need to call eval on the response data from the platform. We can ask for `demisto-py` for the response as a python object.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Content nightly

https://code.pan.run/xsoar/content/-/pipelines/4759375

### CODE REVIEW

**Comment by daryakoval** [comment]
Have you triggered nightly?

**Comment by daryakoval** [comment]
Have you triggered nightly?

**Review by daryakoval — APPROVED** [review]

File: Tests/private_build/tests/test_configure_and_test_integration_instances_private.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ class MockApiClient:
     def __init__(self):
         self.configuration = MockConfiguration()
 
-    def call_api(self, resource_path, method, header_params, files, auth_settings):
+    def call_api(self, resource_path, method, header_params, files, auth_settings, response_type):
```
Comment (daryakoval) [inline Tests/private_build/tests/test_configure_and_test_integration_instances_private.py:39]: No default value for response_type?

Comment (ilaner) [reply Tests/private_build/tests/test_configure_and_test_integration_instances_private.py:39]: This is the mock, this value is ignored anyway

**Pull request merged**